src/cljs/witan/ui/components/data.cljs

```diff
@@ -126,7 +126,7 @@
             [:hr]
             [:div.actions
              [:a {:href (str
-                         "http://"
+                         (if (boolean (cljs-env :witan-api-secure)) "https://" "http://")
```

Comment:
While you're here would you mind changing this to use `:gateway/address` etc from `data/config`? :angel: